src/Examples/DAE/petsc_pid_example.ipynb

```diff
@@ -0,0 +1,776 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "5b2afef3-35bb-4424-8138-d9357d7c5195",
+   "metadata": {},
+   "source": [
+    "# PETSc Time-stepping Solver -- PID Control and Steam Example\n",
+    "\n",
+    "This example provides an overview of the PETSc time-stepping solver utilities in IDAES, which can be used to solve systems of differential algebraic equations (DAEs). PETSc is a solver suite developed primarily by Argonne National Lab (https://petsc.org/release/). IDAES provides a wrapper for PETSc (https://github.com/IDAES/idaes-ext/tree/main/petsc) that uses the AMPL solver interface (https://ampl.com/resources/learn-more/hooking-your-solver-to-ampl/) and utilities functions that allow Pyomo and Pyomo.DAE (https://pyomo.readthedocs.io/en/stable/modeling_extensions/dae.html) problems to be solved using PETSc.\n",
+    "\n",
+    "This demonstration uses the IDAES PID controller model and a flowsheet aranged like so:\n",
+    "\n",
+    "```\n",
+    "             \n",
+    "->--|><|------[]------|><|-->-\n",
+    "   valve_1   tank   valve_2   \n",
+    "```\n",
+    "\n",
+    "where the tank pressure is controlled by the opening of valve_1. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a6d59fd5-b7b5-4e06-9615-99f0538aa627",
+   "metadata": {},
+   "source": [
+    "## Prerequisites\n",
+    "\n",
+    "The PETSc solver is an extra download for IDAES.  The solver is available for Linux, but can be run in the Windows environment via the WSL (Windows Subsystem for Linux).  The WSL method of running the solver works smoothly in the normal Windows workflow.  PETSc installation details for Linux and Windows are available along with the documentation (https://idaes-pse.readthedocs.io/en/latest/technical_specs/core/solvers.html).\n",
+    "\n",
+    "You may want to review the \"PETSc Time-stepping Solver -- Chemical Akzo Nobel Example\" notebook first."
```

Comment:
You might want to change this reference to the other example notebook into a link for clarity.